packages/devtools_app/lib/src/memory/memory_controller.dart

```diff
@@ -48,34 +51,63 @@ class MemoryController {
 
   int selectedSample = -1;
 
-  bool _paused = false;
+  static const String liveFeed = 'Live Feed';
 
-  bool get paused => _paused;
+  /// Notifies that the source of the memory feed has changed.
+  ValueListenable get memorySourceNotifier => _memorySourceNotifier;
 
-  void pauseLiveFeed() {
-    _paused = true;
+  final _memorySourceNotifier = ValueNotifier<String>(liveFeed);
+
+  set memorySource(String source) {
```

Comment:
nit change to a method `selectMemorySource(String source)` so it doesn't feel like UI clients are manually updating the value of memorySource (as that should happen in the controller)